playbooks/roles/airship-deploy-ucp/tasks/main.yml

```diff
@@ -251,28 +251,39 @@
     - install
     - skip_ansible_lint
 
-# TODO(aagate): Add a changed_when: to help idempotency
-- name: Wait until Armada api pod is deployed
-  command: 'kubectl get pod -l application=armada,component=api -n {{ ucp_namespace_name }} -o jsonpath="{.items[0].metadata.name}"'
-  register: armada_results
-  until: armada_results.stdout.find('armada-api-') == 0
-  retries: 180
-  delay: 10
-  tags:
-    - skip_ansible_lint
+- block:
+    # TODO(aagate): Add a changed_when: to help idempotency
+    - name: Wait until Armada api pod is deployed
+      command: 'kubectl get pod -l application=armada,component=api -n {{ ucp_namespace_name }} -o jsonpath="{.items[0].metadata.name}"'
+      register: armada_results
+      until: armada_results.stdout.find('armada-api-') == 0
+      retries: 180
+      delay: 10
+      tags:
+        - skip_ansible_lint
```

Comment:
`changed_when` would be better. With `skip_ansible_lint`, the linter is happy but when you run `ansible-playbook` it will always show you that something changed (but in fact this command only gets data). Using ` changed_when` would fix that...

Reply:
Fix in https://github.com/SUSE-Cloud/socok8s/pull/560